RestSharp/RestRequest.cs

```diff
@@ -368,79 +360,79 @@ public IRestRequest AddObject(object obj, params string[] includedProperties)
         /// <param name="p">Parameter to add</param>
         /// <returns></returns>
         public IRestRequest AddParameter(Parameter p)
         {
-            this.Parameters.Add(p);
+            Parameters.Add(p);
 
             return this;
         }
 
         /// <summary>
-        /// Adds a HTTP parameter to the request (QueryString for GET, DELETE, OPTIONS and HEAD; Encoded form for POST and PUT)
+        ///     Adds a HTTP parameter to the request (QueryString for GET, DELETE, OPTIONS and HEAD; Encoded form for POST and PUT)
         /// </summary>
         /// <param name="name">Name of the parameter</param>
         /// <param name="value">Value of the parameter</param>
         /// <returns>This request</returns>
         public IRestRequest AddParameter(string name, object value)
         {
-            return this.AddParameter(new Parameter
-                                     {
-                                         Name = name,
-                                         Value = value,
-                                         Type = ParameterType.GetOrPost
-                                     });
+            return AddParameter(new Parameter
+            {
+                Name = name,
+                Value = value,
+                Type = ParameterType.GetOrPost
+            });
         }
 
         /// <summary>
-        /// Adds a parameter to the request. There are four types of parameters:
-        /// - GetOrPost: Either a QueryString value or encoded form value based on method
-        /// - HttpHeader: Adds the name/value pair to the HTTP request's Headers collection
-        /// - UrlSegment: Inserted into URL if there is a matching url token e.g. {AccountId}
-        /// - RequestBody: Used by AddBody() (not recommended to use directly)
+        ///     Adds a parameter to the request. There are four types of parameters:
+        ///     - GetOrPost: Either a QueryString value or encoded form value based on method
+        ///     - HttpHeader: Adds the name/value pair to the HTTP request's Headers collection
+        ///     - UrlSegment: Inserted into URL if there is a matching url token e.g. {AccountId}
+        ///     - RequestBody: Used by AddBody() (not recommended to use directly)
         /// </summary>
         /// <param name="name">Name of the parameter</param>
         /// <param name="value">Value of the parameter</param>
         /// <param name="type">The type of parameter to add</param>
         /// <returns>This request</returns>
         public IRestRequest AddParameter(string name, object value, ParameterType type)
         {
-            return this.AddParameter(new Parameter
-                                     {
-                                         Name = name,
-                                         Value = value,
-                                         Type = type
-                                     });
+            return AddParameter(new Parameter
+            {
```

Comment:
Why was there so much weird formatting? I see that you fixed it all over the place.

Reply:
People used tabs. Tabs are evil.